resources/quality/dagoma_pro_430_bowden/Steel nozzle 0.4/dagoma_pro_430_bowden_0.4_abs_high_quality.inst.cfg

```diff
@@ -0,0 +1,58 @@
+[general]
+
+version = 4
+name = High Quality
+definition = dagoma_pro_430_bowden
+
+[metadata]
+
+setting_version = 19
+type = quality
+quality_type = high_quality
+weight = -1
+material = dagoma_generic_abs
+variant = 0.4
+
+[values]
+
+wall_thickness = =machine_nozzle_size * 2
```

Comment:
Don't use machine_nozzle_size for this, since the line width can be different. So this should be =line_width * 2

Take care to change this for all other profiles as well.